DED-564 Update homepage in Docs (#1320)
livechat/livechat-public-docs: src/pages/index.mdx

---
weight: 0
title: "Platform Docs"
tagline: "The source of knowledge of the Platform"

> desc: "Text is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform documentation to find out more."

Author: ```suggestion
desc: "Text enables entrepreneurs to build technology that provides better customer service apps and helps them reach new audiences. Browse the Platform documentation to find out more."
```

> desc: "Text Platform is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform Docs documentation to find out more."

Author: Now I think that changing it to `Platform Docs` would be a better idea🤔 I changed it as the first thing and later forgot about it as it's not visible on the preview.

hidden: false
---

# About Platform Docs

> **Text** is an online customer service software with live support, help desk, and web analytics capabilities. It's used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **Platform Docs** is an ecosystem for developers that grew mainly around two Text products, <a href="https://www.livechat.com/" target="_blank">LiveChat</a> and <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>.

Reviewer: Platform Docs -> Platform (also in some other places)

> **<a href="https://text.com/" target="_blank">Text</a>** enables entrepreneurs to build technology that provides better customer service apps and helps them reach new audiences. Text products are used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **Text Platform** is an ecosystem for developers that grew mainly around two Text products, <a href="https://www.livechat.com/" target="_blank">LiveChat</a> and <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>.

Author: I would add the link to the Text Platform when the website starts

> From the technical perspective the core of Platform Docs is the **open messaging protocol**. The Messaging APIs let you integrate any communication channels directly into LiveChat product. From the business point of view, we're a team dedicated to build vivid [LiveChat Marketplace](https://www.livechat.com/marketplace/) and HelpDesk Marketplace full of apps and services created by <a href="https://developers.livechat.com/" target="_blank">our community of developers</a>.

Author: ```suggestion
From the technical perspective, the core of the Platform is the **open messaging protocol**. The Messaging APIs let you integrate any communication channels directly into LiveChat product. From the business point of view, we're a team dedicated to build vivid [LiveChat Marketplace](https://www.livechat.com/marketplace/) and HelpDesk Marketplace full of apps and services created by <a href="https://developers.livechat.com/" target="_blank">our community of developers</a>.
```

<SectionBanner
  image
  to={"/extending-agent-app/"}

>   title={"Extending App interface"}

Author: ```suggestion
  title={"Extending App Interface"}
```

>   title={"Extending App Interface"}

Reviewer: ```suggestion
  title={"Extending App Interfaces"}
```

>   title={"Extending Apps' interfaces"}

Author: As this section includes SDKs for both products, I decided to change the title here and adjust the description so it mentioned both products.

Reviewer: I'd drop the `'` and make `apps` singular

  desc={

>     "With Platform tools, you can easily extend the LiveChat and HelpDesk app interfaces. Build apps to add new panels, widgets, and other UI elements."

Author: ```suggestion
    "With Platform tools, you can easily extend the LiveChat or HelpDesk app interface. Build apps to add new panels, widgets, and other UI elements."
```

>     "With Platform tools, you can easily extend the LiveChat and HelpDesk app interfaces. Build apps to add new panels, widgets, and other UI elements."

Author: ```suggestion
    "With Platform tools, you can easily extend the LiveChat and HelpDesk app interface. Build apps to add new panels, widgets, and other UI elements."
```

>     "With Platform Docs's tools, you can easily extend the LiveChat and HelpDesk Apps interfaces. Build apps to add new panels, widgets, and other UI elements."

Reviewer: ```suggestion
    "With Platform tools, you can easily extend the LiveChat and HelpDesk app interfaces. Build apps to add new panels, widgets, and other UI elements."
```
